client/components/ZoopListItem.tsx

```diff
@@ -0,0 +1,38 @@
+import { Fave } from "@prisma/client";
+import React from "react";
+import Zoop from "./Zoop";
+
+import Link from "@mui/material/Link";
+
+type ZoopListItemProps = {
+  zoopId: number;
+  author: string;
+  receiver: string;
+  content: string;
+  faves: Fave[];
+  dateCreated: Date;
```

Comment:
Same: can't this just be zoop: Zoop?
```suggestion
  zoop: Zoop
```